UW Templates/Husky Nation (Event).html

```diff
@@ -997,7 +1011,8 @@ <h5 style="font-family:arial,sans-serif,helvetica;font-size:18px;font-weight:bol
             <table border="0" cellspacing="0" cellpadding="0" width="100%" class="anti-spam-row white-background mktoModule" id="copyrightRow" mktoName="Copyright & Address Footer" role="presentation">
                 <tr>
                     <td align="center" valign="top" class="text apple-link-grey" style=" background-color: #ffffff; padding: 5px 15px 10px 15px; font-family: 'Open Sans', arial, sans-serif; font-weight: 400; font-size: 9px; line-height: 2.0; text-align: center; text-transform: none; color: #3d3d3d; -ms-text-size-adjust: none; -webkit-text-size-adjust: none;">
-                    <div class="mktoText" id="copyrightAndAddress" mktoName="Copyright & Address">&copy; 2017 Unit Name and address go here.</div>
+                    <div class="mktoText" id="copyrightAndAddress" mktoName="Copyright & Address">&copy; 2018 Unit Name and address go here.</div>

```

Comment:
@Gonzo427 I wonder if (in a future update) you can generate the current year using Velocity.

Reply:
@nambuben It is possible, but only as a token, as I understand it. Tokens can't be applied to templates, but only to programs.